src/onboarding/Onboarding.js

```diff
@@ -26,16 +26,14 @@ import {
 } from './domain-states'
 
 const SPRING_SHOW = {
-  stiffness: 120,
-  damping: 17,
-  precision: 0.001,
+  tension: 120,
+  friction: 17,
 }
 const SPRING_HIDE = {
-  stiffness: 70,
-  damping: 15,
-  precision: 0.001,
+  tension: 70,
+  friction: 15,
 }
-const SPRING_SCREEN = springConf('slow')
+const SPRING_SCREEN = springs.lazy
```

Comment:
Was playing around with this a bit, and I think `lazy` might be a little bit too slow here and in `StepsBar`. Now the transition fade-in is _really_ obvious and kind of distracting as compared to before.

Reply:
Yeah I was trying to not use the same spring than the action buttons, to create a feel a bit more “organic”, but you’re right it is too slow now.